src/mappening/api/pages.py

```diff
@@ -0,0 +1,91 @@
+from mappening.utils.database import *
```

Comment:
import *'s can get confusing i.e where did this function come from??? Import just what you need explictly, i.e saved_pages_collection

Reply:
Heh sorry made it work then didn't get around to explicit imports